tests/test_content_section.py

```diff
@@ -190,3 +192,23 @@ def test_filtering_a_section_with_a_description_template(self):
 
         assert section.filter({"lot": "digital-specialists"}).description == "description for specialists"
         assert section.filter({"lot": "digital-outcomes"}).description == "description for outcomes"
+
+    def test_question_followup_get_data(self):
+        section = ContentSection(
+            slug='section',
+            name=TemplateField('Section one'),
+            prefill=False,
+            editable=False,
+            edit_questions=False,
+            questions=[Question({'id': 'q1', 'followup': {'q2': [True]}, 'type': 'boolean'}),
```

Comment:
All our tests rely on the follow up value being `True`. May be worth considering having a test that uses `False`. Would make our tests a bit more rigorous, however I'm guessing it's unlikely that we will actually see that in the frameworks repo so fine either way.

Similarly, the YAML seems to allow us to have multiple values that would trigger the follow up question. I believe followups can only be attached to boolean questions so I'm assuming this is just forward planning and we are happy not testing it.